client/signup/steps/woocommerce-install/step-store-address/index.tsx

```diff
@@ -54,6 +55,23 @@ export default function StepStoreAddress( props: WooCommerceInstallProps ): Reac
 		update( WOOCOMMERCE_DEFAULT_COUNTRY, event.target.value );
 	};
 
+	// @todo: Add a general hook to get and update multi-option data like the profile.
+	function updateProfileEmail( email: string ) {
+		const profile_data = get( WOOCOMMERCE_ONBOARDING_PROFILE ) || {};
+
+		profile_data[ 'store_email' ] = email;
+
+		update( WOOCOMMERCE_ONBOARDING_PROFILE, profile_data );
+	}
+
+	function getProfileEmail() {
+		const profile_data = get( WOOCOMMERCE_ONBOARDING_PROFILE ) || {};
+
+		const storeEmail = profile_data[ 'store_email' ];
+
+		return storeEmail || '';
```

Comment:
Could that be combined to this?

```suggestion
		return profile_data['store_email'] || '';
```